core/src/test/java/org/frankframework/receivers/ReceiverTest.java

```diff
@@ -533,6 +538,70 @@ public void run() {
 		);
 	}
 
+
+	@Test
+	void testJmsMessageWithExceptionUntransactedAckModeClientShouldAckMsgWhenRejected() throws Exception {
+		// Arrange
+		configuration = buildConfiguration(null);
+		PushingJmsListener listener = spy(configuration.createBean(PushingJmsListener.class));
+		listener.setTransacted(false);
+		//noinspection removal
+		listener.setJmsTransacted(false);
+		listener.setAcknowledgeMode(JMSFacade.AcknowledgeMode.CLIENT_ACKNOWLEDGE);
+		doReturn(mock(Destination.class)).when(listener).getDestination();
+		doNothing().when(listener).open();
+		doNothing().when(listener).configure();
+
+		createMessagingSource(listener);
+
+		@SuppressWarnings("unchecked")
+		IListenerConnector<jakarta.jms.Message> jmsConnectorMock = mock(IListenerConnector.class);
+		listener.setJmsConnector(jmsConnectorMock);
+		Receiver<jakarta.jms.Message> receiver = setupReceiver(listener);
+
+		Adapter adapter = setupAdapter(receiver, ExitState.ERROR);
+
+		assertEquals(RunState.STOPPED, adapter.getRunState());
+		assertEquals(RunState.STOPPED, receiver.getRunState());
+
+		// start adapter
+		configuration.configure();
+		configuration.start();
+
+		waitWhileInState(adapter, RunState.STOPPED);
+		waitWhileInState(adapter, RunState.STARTING);
+
+		receiver.setMaxDeliveries(1);
```

Comment:
zou dit niet voor de configure gezet moeten worden?

Reply:
Dat zou kunnen als je het netter vindt, dit is niet de enige test waar het wordt gezet na configure. Deze variabele wordt niet gebruikt in configure, maar bij ontvangst van messages, dus daarom dat het in de praktijk geen verschil maakt.

Reply:
Ja ik snap hem. Mochten we er ooit wel iets van maken dat het configure afhankelijk wordt dan gaan we dat niet merken in de tests, maar wel runtime / in de Larva. Vandaar dat ik het altijd netjes vind de setters voor configure aan te roepen :)

Reply:
Dan zal ik het aanpassen in alle tests waarin ik het zie.
😃